rules-reviewed/openjdk11/openjdk8/tests/data/java-removals/Base64Removals.java

```diff
@@ -0,0 +1,20 @@
+package com.test.project;
+
+import sun.misc.BASE64Decoder;
+import sun.misc.BASE64Encoder;
+
+import java.io.IOException;
+
+public class Base64Removals {
+    public static void main(String[] args) {
+        BASE64Encoder base64Encoder = new BASE64Encoder();
+        BASE64Decoder base64Decoder = new BASE64Decoder();
+
+        try {
+            base64Decoder.decodeBuffer("inputString");
+            base64Encoder.encodeBuffer(new byte[]{'a', 'b'});
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+}
```

Comment:
If I replace this file content with:
```
package com.test.project;

import sun.misc.*;

import java.io.IOException;

public class Base64Removals {
    public static void main(String[] args) {
        BASE64Encoder base64Encoder = new BASE64Encoder();

        try {
            base64Decoder.decodeBuffer("inputString");
            base64Encoder.encodeBuffer(new byte[]{'a', 'b'});
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}
```

Then no Hint is being fired. I doubt this has to to with the Rule created in this PR but I'd like to understand/know where the issue is and how to fix it.